CMakeLists.txt

```diff
@@ -331,6 +349,9 @@ if(QUIC_TLS STREQUAL "mitls")
     add_subdirectory(submodules/everest/msquic/msvc/quiccrypto)
 endif()
 
+include(src/tools/toolbase.cmake)
```

Comment:
I'm not sure if cmake supports this, but what if, instead of including this globally, we have a cmakelist.txt in src/tools that includes this, and has all the `add_subdirectory` calls. Then, we only add the src/tools subdirectory if `QUIC_BUILD`TOOLS` is set?

Reply:
Yeah it supports that.